spec/filters/xml_spec.rb

```diff
@@ -418,4 +418,63 @@
       end
     end
   end
+
+  describe "parsing invalid xml" do
+    subject { described_class.new(options) }
+    let(:options) { ({ 'source' => 'xmldata', 'store_xml' => false }) }
+    let(:xmldata) { "<xml> <sample attr='foo' attr=\"bar\"> <invalid> </sample> </xml>" }
+    let(:event) { LogStash::Event.new(data) }
+    let(:data) { { "xmldata" => xmldata } }
+
+    before { subject.register }
+    after { subject.close }
+
+    it 'does not fail (by default)' do
+      subject.filter(event)
+      expect( event.get("tags") ).to be nil
+    end
+
+    context 'strict option' do
+      let(:options) { super.merge({ 'parse_options' => 'strict' }) }
+
+      it 'does fail parsing' do
+        subject.filter(event)
+        expect( event.get("tags") ).to_not be nil
+        expect( event.get("tags") ).to include '_xmlparsefailure'
```

Comment:
Do we document this `tag`? In other plugins such as `grok`, it can be changed/customized.

Reply:
according to the [docs](https://www.elastic.co/guide/en/logstash/current/plugins-filters-xml.html) the tag isn't documented. 
we usually try to keep PRs such as this one minimalistic, so I think you should open an issue as a feature request to be able to configure the failure tag, on top of that (as the feature is delivered) documentation will get improved ...